addon/components/carousel-arrow.js

```diff
@@ -1,32 +1,29 @@
-import Ember from 'ember';
-import layout from '../templates/components/carousel-arrow';
+import Component from 'ember-component';
+import layout    from '../templates/components/carousel-arrow';
+
+import computed from 'ember-computed';
```

Comment:
They are included in ember-cli-shims like this for now. They might change in the future, but for now it is an improvement over importing Ember everywhere, so I suggest using them. Let me know what your decision is.